package.json

```diff
@@ -8,7 +8,7 @@
     "start": "node index.js",
     "start-dev": "pushd minecraftservermanager && ./node_modules/.bin/react-scripts start && popd",
     "build-web": "pushd minecraftservermanager && ./node_modules/.bin/react-scripts build && popd",
-    "test-backend": "./node_modules/mocha/bin/_mocha -u bdd --colors --reporter mochawesome --reporter-options reportDir=./src/tests/results src/test",
+    "test-backend": "./node_modules/mocha/bin/_mocha -u bdd --colors --reporter mochawesome --reporter-options reportDir=./test/results src/test",
```

Comment:
`mocha --recursive --reporter mochawesome --reporter-options reportDir=test/results`
- Using `./---` relative paths breaks on windows apparently, just `mocha` should be fine?
- `-u bdd -colors` are the defaults, so removing them doesn't hurt
- There's nothing in `src/test` anymore
- `--recursive` as tests are in sub-folders now

Reply:
fixed. I don't see a `src/test` in my checkout, if it's still around after merge, I'll just `git rm` it on master directly 😈